.github/FUNDING.yml

```diff
@@ -1,9 +1,9 @@
 # These are supported funding model platforms
 
 github: # Replace with up to 4 GitHub Sponsors-enabled usernames e.g., [user1, user2]
-patreon: FoundryWorkshop # Replace with a single Patreon username
+patreon: mclemente
 open_collective: # Replace with a single Open Collective username
-ko_fi: Forien # Replace with a single Ko-fi username
+ko_fi: mclemente
```

Comment:
Hey, you got the changes from my fork, but you don't need to change this file since I meant to take the project over in case no one showed up to maintain it. I'd probably remove the file altogether unless someone else takes it over.